app/src/main/java/org/koitharu/kotatsu/core/parser/site/AnibelRepository.kt

```diff
@@ -0,0 +1,182 @@
+package org.koitharu.kotatsu.core.parser.site
+
+import org.koitharu.kotatsu.base.domain.MangaLoaderContext
+import org.koitharu.kotatsu.core.exceptions.ParseException
+import org.koitharu.kotatsu.core.model.*
+import org.koitharu.kotatsu.core.parser.RemoteMangaRepository
+import org.koitharu.kotatsu.utils.ext.*
+import java.util.*
+
+class AnibelRepository(loaderContext: MangaLoaderContext) : RemoteMangaRepository(loaderContext) {
+
+	override val source = MangaSource.ANIBEL
+
+	override val defaultDomain = "anibel.net"
+
+	override val sortOrders: Set<SortOrder> = EnumSet.of(
+		SortOrder.UPDATED
+	)
+
+	override suspend fun getList(
+		offset: Int,
+		query: String?,
+		sortOrder: SortOrder?,
+		tag: MangaTag?
+	): List<Manga> {
+		if (!query.isNullOrEmpty()) {
+			return if (offset == 0) search(query) else emptyList()
+		}
+		val page = (offset / 12).inc()
+		val tagPage = offset.inc() //TODO Load another pages
```

Comment:
`offset` or `page`?

Reply:
I don't understand how, but if I specify page instead of tagPage in if (tag != null), then the list will be filled with 12 identical titles (if the filter on the site produced one manga, if for example two titles on the site, then the list will have 24 items), and if I specify tagPage, then there will be no loading to the second page of the filter. So here I suffered quite a lot

Below are the screenshots if I use the val page, which is used to show the entire manga
![Screenshot_1621100672](https://user-images.githubusercontent.com/61558546/118373290-bf21a900-b5be-11eb-8adb-69bb9032d9f7.png)
![Screenshot_1621100674](https://user-images.githubusercontent.com/61558546/118373291-c21c9980-b5be-11eb-88aa-069057350dd0.png)

Reply:
Fixed with `val page = (offset / 12f).toIntUp().inc()`

Reply:
yes, it`s because you got offset < 12.